Gruntfile.js

```diff
@@ -64,6 +64,14 @@ module.exports = function (grunt) {
         }
       }
     },
+    copy: {
+      ngdocs: {
+        cwd: 'components/angular-patternfly/dist/docs',
+        src: ['**'],
+        dest: 'source/_site/ngdocs',
```

Comment:
Jekyll is set to output to source/_site (and it copies everything in `source`), so this should be just `source/ngdocs`.